docs/migration-from-hermione.md

```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
+- Replace all `import ... from "hermione"` and `require("hermione")` to `import ... from "testplane"` and `require("testplane")`;
+- Replace `declare module "hermione"` to `declare module "testplane"`;
```

Comment:
you can merge these two points as "Replace all imports, requires and declarations [examples here]"